docs/python/inference/api_summary.rst

```diff
@@ -1,60 +1,94 @@
 
-===========
-API Summary
-===========
-
-Summary of public functions and classes exposed
-in *ONNX Runtime*.
+===
+API
+===
 
 .. contents::
     :local:
 
-OrtValue
-=========
+API Overview
+============
 
-*ONNX Runtime* works with native Python data structures which are mapped into ONNX data formats :
-Numpy arrays (tensors), dictionaries (maps), and a list of Numpy arrays (sequences).
-The data backing these are on CPU.
+*ONNX Runtime* loads and runs inference on a model in ONNX graph format, or ORT format (for memory and disk constrained environments).
 
-*ONNX Runtime* supports a custom data structure that supports all ONNX data formats that allows users
-to place the data backing these on a device, for example, on a CUDA supported device. This allows for
-interesting *IOBinding* scenarios (discussed below). In addition, *ONNX Runtime* supports directly
-working with *OrtValue* (s) while inferencing a model if provided as part of the input feed.
+The data consumed and produced by the model can be specified and accessed in the way that best matches your scenario.
+
+Load and run a model
+--------------------
+
+InferenceSession is the main class of ONNX Runtime. It is used to load and run an ONNX model,
+as well as specify environment and application configuration options. 
+
+.. code-block:: python
+
+	session = onnxruntime.InferenceSession('model.onnx')
+
+	outputs = session.run([output names], inputs)
+
+ONNX and ORT format models consist of a graph of computations, modeled as operators,
+and implemented as optimized operator kernels for different hardware targets.
+ONNX Runtime orchestrates the execution of operator kernels via `execution providers`.
+An execution provider contains the set of kernels for a specific execution target (CPU, GPU, IoT etc).
+Without any configuration, the inference session runs on CPU. Other execution provides are
+configured using the `providers` parameter.
+
+.. code-block:: python
+
+	session = onnxruntime.InferenceSession(model, providers=['CUDAExecutionProvider', 'CPUExecutionProvider'])
```

Comment:
I would use multiple lines to avoid readers to scroll to see the end of the line.

```
session = onnxruntime.InferenceSession(
       model,
       providers=['CUDAExecutionProvider', 'CPUExecutionProvider'])
```

Reply:
Fixed